File: uri 1134 - Type of Fuel.py
a = g = d = 0
while True:
    num = int(input())
    if num == 1: a += 1
    elif num == 2: g += 1
    elif num == 3: d += 1
    elif num == 4:
        print("MUITO OBRIGADO")
        print(f"Alcool: {a}")
        print(f"Gasolina: {g}")
        print(f"Diesel: {d}")
        break

File: uri 1134 - Type of Fuel.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int a=0,g=0,d=0,i;
    do{
        scanf("%d",&i);
        if(i==1)
            a++;
        else if(i==2)
            g++;
        else if(i==3)
            d++;
    }while(i!=4);
    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\nGasolina: %d\nDiesel: %d\n",a,g,d);

    return 0;
}
